ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/config/tls/client/PicoCliClientTlsOptions.java

```diff
@@ -43,6 +58,13 @@
 
   @Override
   public Optional<ClientTlsTrustOptions> getTrustOptions() {
+    // don't validate here just send options through if present
+    if (caAuthEnabled && knownServersFile == null) {
```

Comment:
can this still be optional? I.e. there'll always be a "tlsenabled", won't there?

Reply:
removed this in betweeny object